src/MongoDB.Driver.Core/Core/Operations/IRetryableOperation.cs

```diff
@@ -97,6 +97,11 @@ public interface IRetryableReadOperation<TResult> : IExecutableInRetryableReadCo
     /// <typeparam name="TResult">The type of the result.</typeparam>
     public interface IRetryableWriteOperation<TResult> : IExecutableInRetryableWriteContext<TResult>
     {
+        /// <summary>
+        /// Gets the write concern for the operation.
+        /// </summary>
+        WriteConcern WriteConcern { get; }
```

Comment:
All classes that implement this interface already had a WriteConcern property so that's why you only see it added to the interface and not to any classes.

Reply:
Ack